CIP-0146/README.md

```diff
@@ -0,0 +1,249 @@
+
+#### Why limit it to purpose 1854 and not allow both HD (1852) and Multi-Signature (1854) keys?
+Mostly due to compatibility across projects and hardware devices as this is how the well established CIP-1854 
+specification describe that it should be done. Also for discovery, it reduces complexity and increases
+performance by only having to scan for a restricted amount of keys.
+ 
+#### Why not encourage and scan any account index, and not just index 0?
+Account indexes is a very useful feature for normal wallets to separate funds, multi-delegation and much more. 
+However, for multisig wallets, they add little value. The idea behind a multisig is that each participant is its own 
+identity, either person or wallet. Having multiple keys in the multisig from the same root key adds false security.
+
+#### But what if I want to utilise the benefits of account separation for fund splitting between the same set of participants?
+This can easily be solved without additional accounts by adding an `after` timelock with the current slot height of 
+the blockchain. This creates a unique multisig wallet (address) even if the rest of the script is identical. This way,
+an infinite number of multisig wallets can be created similar to account separation.
+
+#### What roles should be supported on discovery?
+This specification doesn't restrict discovery to any specific role, depending on the use case of the implementer and 
+additional roles defined in the future through new CIPs. CIP-1854 define role 0 (payment) and role 2 (stake) for 
+script wallets. CIP-105 was created to extend key definition with additional governance roles 3-5.
+
+#### What about key role and index restrictions?
+The main reason for the incrementing key index is to mitigate a possible attack vector where a malicious actor could 
+replay a publicly registered multisig wallet, modifying the script in a nefarious way. When the wallets connected to the
+users multisig key are discovered, both the valid and the invalid multisig registrations would be discovered and the 
+user might interact with the malicious wallet. By only allowing a key to be registered once, this threat can be 
+eliminated. 
+
+Keeping key index in sync for all roles for the same registration makes key handling more manageable.
+
+#### Is types mapping in metadata really necessary?
+It's true that on the wallet side when deriving purpose `1854` keys and scanning for registered wallets, it's known 
+from what path the keys where derived and thus one could figure out the type of native script based on key credential. 
+However, enforcing the transaction to include metadata with metadatum label `1854` and the types mapping make it clear 
+that this is a multisig registration and easier to scan for. 
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] The specification is implemented by three wallet providers or dApps (web applications).
+  - [Eternl Wallet](https://eternl.io)
+  - ...

```

Comment:
```suggestion
- [ ] The specification is implemented by three wallet providers or dApps (web applications).
  - [x] [Eternl Wallet](https://eternl.io)
```
For consistency with other CIPs... for which a growing list of implementors implies that there will be more.  It looks like Eternl is already on the list; please untick if the implementation isn't completed to your satisfaction yet.

Reply:
The implementation is "complete" in Eternl but lack the optional metadata encryption. Can easily be added later on without breaking current implementation so I would classify Eternl as ready in this regard. So I will accept your suggested change.